docs/docs/kubernetes/operator/design/design_compliance_report.md

```diff
@@ -0,0 +1,770 @@
+# Support Compliance Reports
```

Comment:
we left compliance out of this release, so remove this

Reply:
Removed